feature/danggn/src/main/java/com/mashup/feature/danggn/ranking/DanggnRankingViewModel.kt

```diff
@@ -32,6 +33,7 @@ class DanggnRankingViewModel @Inject constructor(
 ) : BaseViewModel() {
     companion object {
         private const val DEFAULT_SHAKE_NUMBER = -1
+        private const val DATE_FORMAT = 60 * 60 * 24 * 1000
```

Comment:
```suggestion
        private const val DATE_UNIT = 60 * 60 * 24 * 1000
```

UNIT으로 네이밍하는건 어때요?? :) Format은 "yyyy-mm-dd" 이런 느낌이라서

Reply:
[acca6fb](https://github.com/mash-up-kr/mashup_Android/pull/421/commits/acca6fbd128a2bdafa917d8cac677628a764d29c) 리뷰 반영